tests/php/Configuration/Parser/ContentTypesParserTest.php

```diff
@@ -69,7 +69,7 @@ public function testHasConfig(): void
         $this->assertCount(6, $config);
 
         $this->assertArrayHasKey('homepage', $config);
-        $this->assertCount(21, $config['homepage']);
+        $this->assertCount(20, $config['homepage']);
```

Comment:
What does this magic number mean? Why it has to be 20?

Reply:
Because it happens to have 20 items.

Reply:
But must it be 20 items? In other words, this value is special enough to test it?

Reply:
Currently, yes. It should be exactly 20, because that is the number of attributes a ContentType has. If any are missing from the config yaml, they will be set to the default.

Reply:
So can you set it as const in ContentType class?